roles/deploy/hooks/finalize-after.yml

```diff
@@ -13,6 +13,22 @@
     chdir: "{{ deploy_helper.new_release_path }}"
   when: wp_installed.rc == 0
 
+- name: Update WP database
+  command: wp core update-db
+  args:
+    chdir: "{{ deploy_helper.new_release_path }}"
+  when: wp_installed.rc == 0 and not (project.multisite.enabled | default(False) == True)
```

Comment:
oh and the parentheses aren't need either but it might be a subject thing:

`wp_installed | success and not project.multisite.enabled | default(false)`